### DISCUSSION TITLE
Is UnitsNet based on an industry standard?

### DISCUSSION THREAD

**Discussion by Brunni** [angularsen/UnitsNet#1123] [Q&A]
Hi,
is the library based on some industry standard or are conversions to common standards available?

Standards I know so far:
* https://en.wikipedia.org/wiki/International_System_of_Units
* Common Code CEFACT https://unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex2e.pdf

Best regards

**Comment by angularsen** [comment]
No, we haven't strictly followed any standards, but if such complete works exist and is publicly available then that would be useful to use as reference for naming and conversion factors.

I like that PDF and how it lists both abbreviations and conversion factors to SI units, which should more or less match what our setup for most units.

I added this as an example reference to the wiki:
https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit#1-add-or-modify-json-file-for-a-quantity-class

I'll keep it in mind when reviewing future additions of new units, but I don't currently have the time to go and verify all the existing units against this reference.

**Reply by Brunni** [reply]
The PDF is also available as excel:
https://tfig.unece.org/contents/recommendation-20.htm
Sadly the link there is HTTP so it does not open directly. You have to add the s :)
https://www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_Rev7e_2010.zip

**Reply by Brunni** [reply]
And as further source there is this wikipedia page as well: https://en.wikipedia.org/wiki/Unified_Code_for_Units_of_Measure
Never heard about UCUM before: https://ucum.org/
